priam/src/main/java/com/netflix/priam/config/IConfiguration.java

```diff
@@ -1091,4 +1091,14 @@ default String getMergedConfigurationDirectory() {
      * @return The result for the property, or the defaultValue if provided (null otherwise)
      */
     String getProperty(String key, String defaultValue);
+
+    /**
+     * Returns a csv of backup component file types {@link
```

Comment:
In the comment can you explain the default value of StringUtils.EMPTY? Also, provide an example of override value.